src/common/libs/Config/configStructureArray.php

```diff
@@ -130,6 +172,27 @@
     "default" => false,
     "envFallback" => "bCMS__FROM_EMAIL",
   ],
+  "EMAILS_FROMDOMAIN" => [
```

Comment:
Could we not just grab this from the from email?

Reply:
Or is it like a config thing like domain id 182627

Reply:
Thought it was specific, turns out we can!